compiler/runtime/OMRCodeCacheManager.cpp

```diff
@@ -125,13 +125,13 @@ OMR::CodeCacheManager::initialize(
    config._needsMethodTrampolines =
       !(config.trampolineCodeSize() == 0
         || config.maxNumberOfCodeCaches() == 1
-#if !defined(STRESS_TRAMPOLINES) && !defined(TR_HOST_POWER)
+#if !defined(STRESS_TRAMPOLINES) && !defined(TR_HOST_POWER) && !defined(TR_TARGET_64BIT)  
```

Comment:
The extra 32 bit check is added because when I was doing personal build tests the 32 bit tests would fail when stress trampoline is defined as they do not have trampoline code. I added the check so that running personal builds would be easier in the future. Should I remove it?